libreantdb/test/test_iterate.py

```diff
@@ -0,0 +1,47 @@
+'''
+This module will connect to your elasticsearch instance.
+An index will be reserved to the tests.
+'''
+
+from nose.tools import ok_, eq_, with_setup
+
+from . import db, cleanall
+
+
+@with_setup(cleanall, cleanall)
+def test_iterate_all():
+    n = 5
+    ids = list()
+    for i in range(n):
+        id = db.add_book(body=dict(num=i, title='Un libro', _language='it'))['_id']
+        ids.append(id)
+    db.es.indices.refresh(index=db.index_name)
+    books = [b for b in db.iterate_all()]
+    for book in books:
+        ok_(book['_id'] in ids)
```

Comment:
On Tue, Jun 09, 2015 at 12:33:04PM -0700, ael wrote:
>if all ids of `books` are in `ids` and `book` has the same length of
>`ids` we can assume that the two list have the same element. Isn't it?

it is not. If `ids=[1,2,3]` and `books=[1,1,1]` the test passes, but it is
obviously false.